api/envoy/config/rbac/v2/BUILD

```diff
@@ -14,13 +21,13 @@ api_proto_library_internal(
     ],
 )
 
-api_go_proto_library(
-    name = "rbac",
-    proto = ":rbac",
-    deps = [
-        "//envoy/api/v2/core:address_go_proto",
-        "//envoy/api/v2/route:route_go_proto",
-        "//envoy/type/matcher:metadata_go_proto",
-        "//envoy/type/matcher:string_go_proto",
-    ],
-)
+#api_go_proto_library(
```

Comment:
?

Reply:
This is terrible in Go. This is because https://github.com/googleapis/go-genproto and http://github.com/google/googleapis cause an ODR violation. It might be possible to expose more protos in googleapis, but I'm not holding my breath there.

Reply:
Let me try to push googleapis change from within and get this resolved. Otherwise, we need to do importmaps (e.g. https://github.com/envoyproxy/go-control-plane/blob/master/build/generate_protos.sh#L27). Is this something you would like to see in bazel build here?

Reply:
Fixed